core/src/main/java/org/apache/iceberg/PartitionData.java

```diff
@@ -35,7 +35,7 @@
 import org.apache.iceberg.types.Type;
 import org.apache.iceberg.types.Types;
 
-class PartitionData
+public class PartitionData
```

Comment:
I had to make this public as `TableScanUtil` is in another package and I needed a serializable holder for partition data.